app/controllers/api/v1/slack_controller.rb

```diff
@@ -0,0 +1,19 @@
+class Api::V1::SlackController < Api::V1::BaseController
+  def get_users
```

Comment:
Assignment Branch Condition size for get_users is too high. [27.82/25]<br>Cyclomatic complexity for get_users is too high. [7/6]